testing/get_http_bound_address.py

```diff
@@ -0,0 +1,7 @@
+#
+# get_http_bound_address.py
+# Mich, 2016-02-05
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+mldb.log(mldb.get_http_bound_address()) # noqa
```

Comment:
can we find some appropriate test to attach this to instead of creating a whole new test?